src/_gettsim_tests/test_renten_alter.py

```diff
@@ -0,0 +1,145 @@
+import itertools
+
+import numpy as np
+import pandas as pd
+import pytest
+from _gettsim.interface import compute_taxes_and_transfers
+from _gettsim.policy_environment import set_up_policy_environment
+from pandas.testing import assert_series_equal
+
+from _gettsim_tests import TEST_DATA_DIR
+
+INPUT_COLS = [
+    "p_id",
+    "hh_id",
+    "tu_id",
+    "alter",
+    "jahr",
+    "geburtsjahr",
+    "geburtsmonat",
+    "m_arbeitsunfähig",
+    "m_krank_ab_16_bis_24",
+    "m_mutterschutz",
+    "m_arbeitslos",
+    "m_ausbild_suche",
+    "m_schul_ausbild",
+    "m_alg1_übergang",
+    "m_geringf_beschäft",
+    "weiblich",
+    "y_pflichtbeitr_ab_40",
+    "m_pflichtbeitrag",
+    "m_freiw_beitrag",
+    "m_ersatzzeit",
+    "m_kind_berücks_zeit",
+    "m_pfleg_berücks_zeit",
+]
+
+OUT_COLS = [
+    "ges_rente_regelaltersgrenze",
+    "ges_rente_frauen_altersgrenze",
+    "_ges_rente_langj_altersgrenze",
+    "_ges_rente_besond_langj_altersgrenze",
+]
+
+YEARS = [2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021]
+
+
+@pytest.fixture(scope="module")
+def input_data():
+    # Original Data Source: Seibold, A. (2021).
+    # Reference points for retirement behavior:
+    # Evidence from german pension discontinuities.
+    # American Economic Review, 111(4), 1126-65.
+    # Data: https://www.openicpsr.org/openicpsr/project/120903/version/V1/view
+    # retirement ages wide.dta
+
+    # IDs were added to the original data
+    original_data = pd.read_csv(TEST_DATA_DIR / "renten_alter.csv")
+
+    original_data.columns = original_data.columns.str.lower()
+
+    # Rename variables to GETTSIM convention
+    var_names = {
+        "id": "p_id",
+        "yob": "geburtsjahr",
+        "mob": "geburtsmonat",
+        "regsray": "Regelaltersgrenze_Jahr",
+        "regsram": "Regelaltersgrenze_Monat",
+        "ltsray": "Langjährig_Jahr",
+        "ltsram": "Langjährig_Monat",
+        "wosray": "Rente_Frauen_Jahr",
+        "wosram": "Rente_Frauen_Monat",
+        "vltsray": "Besonders_Langjährig_Jahr",
+        "vltsram": "Besonders_Langjährig_Monat",
+    }
+    out = original_data[[*var_names]].rename(columns=var_names).copy()
+
+    # Create IDs
+    out["tu_id"] = out["p_id"]
+    out["hh_id"] = out["p_id"]
+
+    # Create variables needed in GETTSIM
+    out["jahr"] = 2010
```

Comment:
Not sure if  I am understanding the test logic correctly but if we ware giving it the year 2010 and the very long term insured threshold 63 for cohort 1947 then there should be a bug because back then it was still 65 -- and that is what Gettsim would tell us. If we instead put 2015 - the 63 is valid and should work.

Reply:
I checked whether it works for 2015, however, the same tests keep failing. 
In the original test file there are no values given for cohorts born before 1947. The problem is that GETTSIM calculates 63 for those cohorts when running the tests for years 2015 and later, while when running the tests for the years before 2015 GETTSIM calculates 65. So I don't really know to what value I should set the input, since I don't want to change the original file manually

Reply:
This column should not have an effect. It is also no longer required by GETTSIM recently. So we can delete the line of code. The used policy parameters and functions do only depend on the date given to `set_up_policy_environment`

```suggestion
```

Reply:
> 
Isn't there a possibility that the tests based on this data column are only run for 2015 onwards?

Reply:
Which data column do you mean exactly? For most of the other test cases the input data contains a column "jahr" and each policy-year is only checked for the rows in the input data with the respective year

Reply:
vltsray and vltsram

Reply:
But we could just map that to the year passed to setup_policy_environment and do not need a column in the data, right?

Reply:
yeah, thats basically what I though of , to avoid manipulating the input data file

Reply:
That is, it is the same for everybody, or am I mistaken?

Reply:
?

Reply:
Sorry, our two messages crossed there. We just do not need that variable, I believe. Unless I am misinterpreting what Arthur's dataset is supposed to do, see below. https://github.com/iza-institute-of-labor-economics/gettsim/pull/428#issuecomment-1406520901

Let's continue down there.